src/crawlee/cli.py

```diff
@@ -0,0 +1,83 @@
+# ruff: noqa: FA100 ASYNC100
+import asyncio
+from functools import wraps
+from typing import Annotated, Any, Callable, Coroutine, Union
+
+import httpx
+import inquirer
+import typer
+from cookiecutter.main import cookiecutter
+
+TEMPLATE_LIST_URL = 'https://api.github.com/repos/apify/crawlee-python/contents/templates?ref=project-templates'  # TODO remove the ?ref before merging
+
+
+def run_async(func: Callable[..., Coroutine]) -> Callable:
+    """Decorates a coroutine function so that it is ran with `asyncio.run`."""
+
+    @wraps(func)
+    def wrapper(*args: Any, **kwargs: Any) -> None:
+        asyncio.run(func(*args, **kwargs))
+
+    return wrapper
+
+
+cli = typer.Typer()
+
+
+@cli.callback()
+def callback() -> None:
+    """An empty callback to force typer into making a CLI with a single command."""
+
+
+@cli.command()
+@run_async
+async def create(
+    project_name: Annotated[
+        Union[str, None],
+        typer.Argument(
+            help='The name of the project and the directory that will be created to contain it. '
+            'If none is given, you will be prompted.'
+        ),
+    ] = None,
+    template: Annotated[
+        Union[str, None],
+        typer.Option(help='The template to be used to create the project. If none is given, you will be prompted.'),
+    ] = None,
+) -> None:
+    """Bootstrap a new Crawlee project."""
+    if template is None:
+        templates_response = httpx.get(TEMPLATE_LIST_URL)
+        template_choices: list[str] = [item['name'] for item in templates_response.json() if item['type'] == 'dir']
+    else:
+        template_choices = []
+
+    answers = (
+        inquirer.prompt(
+            [
+                inquirer.Text(
+                    'project_name',
+                    message='Name of the new project folder',
+                    validate=lambda _, it: len(it) > 0,
+                    ignore=project_name is not None,
+                ),
+                inquirer.List(
+                    'template',
+                    message='Please select the template for your new Crawlee project',
+                    choices=[(choice[0].upper() + choice[1:], choice) for choice in template_choices],
```

Comment:
This is not as pretty as a handwritten description, but we don't have to maintain a manifest. It sounds like an OK tradeoff to me, but I'm willing to discuss this.